services/net/syndication/SyndicationAction.cs

```diff
@@ -260,6 +260,18 @@ private SourceContent CreateSourceContent(IngestModel ingest, SyndicationItem it
         var contentType = ingest.IngestType?.ContentType ?? throw new InvalidOperationException($"Ingest '{ingest.Name}' is missing ingest content type.");
         var publishedOn = item.PublishDate.UtcDateTime != DateTime.MinValue ? item.PublishDate.UtcDateTime : (DateTime?)null;
         var uid = Runners.BaseService.GetContentHash(source, title, publishedOn);
+        string? media = null;
+
+        // Extract values from namespaces.  I don't know a better way to do this.
+        foreach (var ext in item.ElementExtensions)
```

Comment:
I really don't understand why the default Microsoft SyndicationFeed doesn't know how to use namespaces.  I looked for a few hours for some example code and couldn't find any.